core/src/main/java/com/google/cloud/sql/core/Connector.java

```diff
@@ -118,6 +119,10 @@ Socket connect(ConnectionConfig config, long timeoutMs) throws IOException {
       SSLSocket socket = (SSLSocket) metadata.getSslContext().getSocketFactory().createSocket();
       socket.setKeepAlive(true);
       socket.setTcpNoDelay(true);
+
+      SSLParameters sslParams = new SSLParameters();
+      socket.setSSLParameters(sslParams);
```

Comment:
can you add a comment in the code here explaining this new piece?

Reply:
Removed. This was unnecessary.